.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+## Description
+
+*Replace this paragraph with a description of what this PR is doing. If you're
+modifying existing behavior, describe the existing behavior, how this PR is
+changing it, and what motivated the change.
+
+## Related Issues
+
+*Replace this paragraph with a list of issues related to this PR from our [issue
+database]. Indicate, which of these issues are resolved or fixed by this PR.
+There should be at least one issue listed here.*
+
+## Tests
+
+I added the following tests:
+
+*Replace this with a list of the tests that you added as part of this PR. A
+change in behaviour with no test covering it will likely get reverted
+accidentally sooner or later. PRs must include tests for all
+changed/updated/fixed behaviors. See [testing the engine] for instructions on
+writing and running engine tests.*
+
+## Checklist
```

Comment:
This is not part of the commit message right? I don't think having the same completed checklist over and over in the commit log adds value to the log. If its just part of the GitHub comment, then thats fine.

Reply:
No. The squash commit message is unrelated to the pull request comment history.

Reply:
Sounds good.